[2024-03-28T07:34:14.934+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingesting_data_bq.local_to_gcs_0_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2024-03-28T07:34:15.015+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingesting_data_bq.local_to_gcs_0_task scheduled__2024-03-27T00:00:00+00:00 [queued]>
[2024-03-28T07:34:15.018+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-28T07:34:15.126+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): local_to_gcs_0_task> on 2024-03-27 00:00:00+00:00
[2024-03-28T07:34:15.181+0000] {standard_task_runner.py:60} INFO - Started process 320 to run task
[2024-03-28T07:34:15.347+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ingesting_data_bq', 'local_to_gcs_0_task', 'scheduled__2024-03-27T00:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_to_gcs.py', '--cfg-path', '/tmp/tmpyev2k6st']
[2024-03-28T07:34:15.376+0000] {standard_task_runner.py:88} INFO - Job 408: Subtask local_to_gcs_0_task
[2024-03-28T07:34:15.908+0000] {task_command.py:423} INFO - Running <TaskInstance: ingesting_data_bq.local_to_gcs_0_task scheduled__2024-03-27T00:00:00+00:00 [running]> on host d3d7fa608a49
[2024-03-28T07:34:16.385+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingesting_data_bq' AIRFLOW_CTX_TASK_ID='local_to_gcs_0_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T00:00:00+00:00'
[2024-03-28T07:34:21.316+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-28T07:34:21.329+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=ingesting_data_bq, task_id=local_to_gcs_0_task, execution_date=20240327T000000, start_date=20240328T073414, end_date=20240328T073421
[2024-03-28T07:34:21.505+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-28T07:34:21.563+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
